### PR TITLE
Release announcement for 0.13.0

### PR DESCRIPTION
**Pull request by qiujian16**

### CODE REVIEW

**Comment by zhujian7** [comment]
I think we should mention that when upgrading addon-framework to 0.9.0, [`patch` `clustermanagementaddon` permission should be added](https://github.com/open-cluster-management-io/addon-framework/pull/236#discussion_r1475478681).

File: content/en/community/releases.md [hunk 1]
```diff
@@ -3,9 +3,42 @@ title: Releases
...
+  and will be also used in policy addon. The API provides user a way to define workload/addon rolling upgrade strategy
+  across multiple clusters. See [here](https://open-cluster-management.io/concepts/addon/#add-on-rollout-strategy) for more
+  details.
+- More install configurations in Klusterlet: we enhance the `Klusterlet` API by adding more configuration field, including
+  resource request, QPS/Burst and priority class.
+- Cloudevent support: an expeiment feature to wire work agent with a messasge broker using cloudevent protocol. This enables
+  work agent running in a high scalable mode. See [here](
+  https://github.com/open-cluster-management-io/enhancements/tree/main/enhancements/sig-architecture/224-event-based-manifestwork)
+  for more details.
+- Addon-framework is upgraded to v0.9.0 to better support generic addon-manager and addon rolling out strategy. It is highly
```
Comment (zhujian7) [inline content/en/community/releases.md:24]: should we mention that `patch clustermanagementaddon permisson` should be added after upgrading to 0.9.0`?

File: content/en/community/releases.md [hunk 2]
```diff
@@ -3,9 +3,44 @@ title: Releases
...
+  recommanded for addons to upgrade the addon-framework depdency v0.8.1 or higher version. Also it should be noted that
+  RBAC update of the addon hub controller (addin PATCH verson for resource `clustermanagementaddons`) is needed during
+  the upgrade.
```
Comment (mikeshng) [inline content/en/community/releases.md:27]:
```suggestion
  the upgrade. 
  - Application add-on `Subscription` API can now aggregate and report managed clusters' kustomization errors.
```

**Review by mikeshng — COMMENTED** [review]
Just want to highlight an app addon feature. The rest LGTM.

File: content/en/community/releases.md [hunk 3]
```diff
@@ -3,9 +3,45 @@ title: Releases
...
 
+## `0.13.0`, 6 Mar. 2024
+
+The Open Cluster Management team is proud to announce the release of OCM v0.13.0! There are a bunch of new features 
+added into this release.
+
+- Rollout API: we build a common rollout API that has beed adopted in `ClusterManagementAddon` and `ManifestWorkReplicaSet`,
+  and will be also used in policy addon. The API provides user a way to define workload/addon rolling upgrade strategy
+  across multiple clusters. See [here](https://open-cluster-management.io/concepts/addon/#add-on-rollout-strategy) for more
+  details.
```
Comment (dhaiducek) [inline content/en/community/releases.md:17]:
```suggestion
- Rollout API: we built a common rollout API that has been adopted in `ClusterManagementAddon` and `ManifestWorkReplicaSet`,
  and will be also used in the policy addon in a future release. The API provides users a way to define a workload/addon rolling upgrade strategy
  across multiple clusters. See [here](https://open-cluster-management.io/concepts/addon/#add-on-rollout-strategy) for more
  details.
```

File: content/en/community/releases.md [hunk 4]
```diff
@@ -3,9 +3,45 @@ title: Releases
...
+- More install configurations in Klusterlet: we enhance the `Klusterlet` API by adding more configuration field, including
+  resource request, QPS/Burst and priority class.
```
Comment (dhaiducek) [inline content/en/community/releases.md:19]:
```suggestion
- More install configurations in Klusterlet: we enhanced the `Klusterlet` API by adding more configuration fields, including
  resource requests, QPS/Burst and priority class.
```

File: content/en/community/releases.md [hunk 5]
```diff
@@ -3,9 +3,45 @@ title: Releases
...
+- Cloudevent support: an expeiment feature to wire work agent with a messasge broker using cloudevent protocol. This enables
+  work agent running in a high scalable mode. See [here](
+  https://github.com/open-cluster-management-io/enhancements/tree/main/enhancements/sig-architecture/224-event-based-manifestwork)
+  for more details.
```
Comment (dhaiducek) [inline content/en/community/releases.md:23]:
```suggestion
- Cloudevent support: an experimental feature to wire the work agent with a message broker using the cloudevent protocol. This enables a
  work agent running in a highly scalable mode. See [here](
  https://github.com/open-cluster-management-io/enhancements/tree/main/enhancements/sig-architecture/224-event-based-manifestwork)
  for more details.
```

File: content/en/community/releases.md [hunk 6]
```diff
@@ -3,9 +3,45 @@ title: Releases
...
+- Addon-framework is upgraded to v0.9.0 to better support generic addon-manager and addon rolling out strategy. It is highly
+  recommanded for addons to upgrade the addon-framework depdency v0.8.1 or higher version. Also it should be noted that
+  RBAC update of the addon hub controller (addin PATCH verson for resource `clustermanagementaddons`) is needed during
+  the upgrade.
```
Comment (dhaiducek) [inline content/en/community/releases.md:27]:
```suggestion
- Addon-framework is upgraded to v0.9.0 to better support the generic addon-manager and addon rolling out strategy. It is highly
  recommended for addons to upgrade the addon-framework depdency to version v0.8.1 or higher. Also it should be noted that an
 RBAC update of the addon hub controller (adding PATCH for resource `clustermanagementaddons`) is required during
  the upgrade. 
```

File: content/en/community/releases.md [hunk 7]
```diff
@@ -3,9 +3,45 @@ title: Releases
...
+- Cloudevent support: an expeiment feature to wire work agent with a messasge broker using cloudevent protocol. This enables
+  work agent running in a high scalable mode. See [here](
+  https://github.com/open-cluster-management-io/enhancements/tree/main/enhancements/sig-architecture/224-event-based-manifestwork)
+  for more details.
+- Addon-framework is upgraded to v0.9.0 to better support generic addon-manager and addon rolling out strategy. It is highly
+  recommanded for addons to upgrade the addon-framework depdency v0.8.1 or higher version. Also it should be noted that
+  RBAC update of the addon hub controller (addin PATCH verson for resource `clustermanagementaddons`) is needed during
+  the upgrade. 
+- Application add-on `Subscription` API can now aggregate and report managed clusters' kustomization errors.
+
```
Comment (dhaiducek) [inline content/en/community/releases.md:29]: @mprahl Do you want to mention the Compliance History API here or do we need to make sure all the pieces are in place in this release before announcing it anywhere?

```suggestion
- New OperatorPolicy to handle OLM operators: A new OperatorPolicy is added, managed by the `config-policy-controller` Pod, aimed at easing the management of OLM deployments.
- Diff logging for ConfigurationPolicy: A `recordDiff` parameter is added to `ConfigurationPolicy` to enable the diff between the object on the cluster and the policy definition to be logged in the `config-policy-controller` Pod logs.

```

**Review by dhaiducek — CHANGES_REQUESTED** [review]
Thanks! Sorry for my belated review. I've added a couple bullets for Policy as well as suggested some other spelling/wording edits.

File: content/en/community/releases.md (hunk 7 above)
Comment (mprahl) [reply content/en/community/releases.md:33]: @dhaiducek I think we can announce it here, but I'll create an issue to add documentation in OCM-io on it.

Comment (dhaiducek) [reply content/en/community/releases.md:33]: Thanks--compliance history added!

File: content/en/community/releases.md [hunk 8]
```diff
@@ -3,9 +3,49 @@ title: Releases
...
+  across multiple clusters. See [here](https://open-cluster-management.io/concepts/addon/#add-on-rollout-strategy) for more
+  details.
+- More install configurations in Klusterlet: we enhanced the `Klusterlet` API by adding more configuration fields, including
+  resource requests, QPS/Burst and priority class.
+- Cloudevent support: an experimental feature to wire the work agent with a message broker using the cloudevent protocol. This enables
+  a work agent running in a highly scalable mode. See [here](
+  https://github.com/open-cluster-management-io/enhancements/tree/main/enhancements/sig-architecture/224-event-based-manifestwork)
+  for more details.
+- Addon-framework is upgraded to v0.9.0 to better support the generic addon-manager and addon rolling out strategy. It is highly
+  recommended for addons to upgrade the addon-framework depdency to version v0.8.1 or higher. Also, it should be noted that
```
Comment (dhaiducek) [inline content/en/community/releases.md:25]:
```suggestion
  recommended for addons to upgrade the addon-framework dependency to version v0.8.1 or higher. Also, it should be noted that
```

**Review by mikeshng — APPROVED** [review]
/approve

/lgtm

**Pull request merged**